libobs/obs-encoder.c

```diff
@@ -1085,13 +1085,14 @@ static inline void push_back_audio(struct obs_encoder *encoder,
 				   struct audio_data *data, size_t size,
 				   size_t offset_size)
 {
+	if (offset_size >= size)
+		return;
 	size -= offset_size;
```

Comment:
To provide a distinct separation, I'd prefer a blank line between the `if` contents and the following line, or else wrap the `if` contents in curly braces.